apps/mark-scan/frontend/src/pages/empty_ballot_box_page.tsx

```diff
@@ -34,7 +36,10 @@ function ConfirmBallotBoxEmptied(): JSX.Element {
       <ConfirmButtonWrapper>
         <Button
           variant="primary"
-          onPress={() => confirmBallotBoxEmptiedMutation.mutate()}
+          onPress={async () => {
+            await logger.log(LogEventId.BallotBoxEmptied, 'poll_worker');
+            confirmBallotBoxEmptiedMutation.mutate();
+          }}
```

Comment:
Maybe log this on the backend? I'm imagining a world in the future where we might be able to get rid of frontend logging completely, since all the important user actions are handled on the backend. Plus, it's probably better to only log it if it actually succeeds

Reply:
I think @carolinemodic mentioned we might want to log button presses on the frontend for actions that require poll worker intervention. Is that still the case? If not, we can delete this entirely because the automatic state transition logging will cover this case.

Reply:
We are supposed to log any action a user (poll worker, election admin) takes and whether or not it succeed/fails

Reply:
[d9675c1](https://github.com/votingworks/vxsuite/pull/4592/commits/d9675c13f4c6f957dc2386af22b9af507abcc8d8)